src/Log/Engine/ConsoleLog.php

```diff
@@ -74,6 +79,10 @@ public function __construct(array $config = [])
         if (isset($config['outputAs'])) {
             $this->_output->setOutputAs($config['outputAs']);
         }
+
+        if ($this->formatter && !isset($this->_config['formatter']['dateFormat'])) {
```

Comment:
This provides backwards compatibility where loggers don't have the dateFormat moved to the formatter config.